README.md

```diff
@@ -42,7 +42,7 @@ A Pokémon is considered evolved if it is the back side of a card with an evolve
 
 **Q: The DFC Kadabra has an ability that triggers when it enters the battlefield. That can't happen, so why is it there?**
 
-**A**: We want to be consistent with the DFC cards having the same abilities on the front and back sides. This also opens up potential design space for future generations if we want to have an effect that has cards enter already evolved.
+**A**: We want to be consistent with the DFC cards having the same abilities on the front and back sides. It can also happen if you were to make a copy of Kadabra.
```

Comment:
This seems like a nonsequitor, maybe _Keep in mind that token copies of Kadabra *would* enter the battlefield_?

Reply:
I actually think we should just take this whole question out tbh. It's pretty rare that someone is even going to notice that it's weird, and those nerds should be able to work out how token copies behave.

Reply:
that sounds good to me